app/views/layouts/application.html.erb

```diff
@@ -45,10 +26,10 @@
           <span class="icon-bar"></span>
           <span class="icon-bar"></span>
         </button>
-        <a class="navbar-brand" href="#">TrainHard</a>
+        <a class="navbar-brand" href="#">StockTracker</a>
         <div class="navbar-collapse collapse navbar-responsive-collapse">
           <ul class="nav navbar-nav">
-            <li><%= link_to "Link1", "/path1"  %></li>
+            <li><%= link_to "Jajo", "/path1"  %></li>
             <li><%= link_to "Link2", "/path2"  %></li>
             <li><%= link_to "Link3", "/path3"  %></li>
```

Comment:
this links here seems not to be valid, try not to commit broken code. Please add routes to handle this or change the paths to `#` or something that won't cause app error

Reply:
corrected!